packages/foundry-sdk-generator/src/generate/betaClient/getModuleSourceFile.ts

```diff
@@ -15,20 +15,38 @@
  */
 
 import type { Project, StringLiteral } from "ts-morph";
+import { DEBUG } from "./minifyBundle.js";
 
-export function getModuleSourceFile(project: Project, node: StringLiteral) {
+export const splitExtension = /^(.*)(\.[cm]?js)$/;
+
+export function getModuleSourceFile(
+  project: Project,
+  node: StringLiteral | string,
+) {
   let exportSourceFile;
+
+  const literalText = typeof node === "string" ? node : node.getLiteralText();
+
+  const withoutExtension = literalText.match(splitExtension)?.[1]
+    ?? literalText;
+
   try {
-    exportSourceFile = project.getSourceFile(`/${node.getLiteralText()}.ts`);
+    exportSourceFile = project.getSourceFile(`/${withoutExtension}.ts`);
     if (!exportSourceFile) {
       exportSourceFile = project.getSourceFile(
-        `/${node.getLiteralText()}/index.ts`,
+        `/${withoutExtension}/index.ts`,
       );
-      if (!exportSourceFile) {
+      if (DEBUG && !exportSourceFile) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `Could not find source file for ${withoutExtension}`,
```

Comment:
This is only a problem if the rolled up code still needs this type. If the type would be elided then its not an issue. Thats why this is a debug print out only. If we are inadvertently not including something that should be there, the test code will catch it.